clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -49,8 +54,10 @@
 public class ConfigDef {
```

Comment:
This class is basically becoming public. Does it make it into Javadocs? If we expect connector developers to use it we should make sure there's sufficient documentation available.